lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -335,20 +316,15 @@ def test_enrollment_cap(self):
         self.assertNotContains(resp, REG_STR)
 
 
+@override_waffle_flag(COURSE_HOME_USE_LEGACY_FRONTEND, active=True)
 class AboutWithInvitationOnly(SharedModuleStoreTestCase):
     """
     This test case will check the About page when a course is invitation only.
     """
-    MODULESTORE = TEST_DATA_MONGO_AMNESTY_MODULESTORE
-
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
         cls.course = CourseFactory.create(metadata={"invitation_only": True})
-        cls.about = ItemFactory.create(
```

Comment:
Correct - there is always an "about" xblock made already for you in Split - so this was erroring out when adding a second of the same type. Since this didn't have any data, it could just be removed. Same as the default about block.